dirs/dirs.go

```diff
@@ -457,7 +458,8 @@ func SetRootDir(rootdir string) {
 	XdgRuntimeDirGlob = filepath.Join(XdgRuntimeDirBase, "*/")
 
 	CompletionHelperInCore = filepath.Join(CoreLibExecDir, "etelpmoc.sh")
-	CompletersDir = filepath.Join(rootdir, "/usr/share/bash-completion/completions/")
+	LegacyCompletersDir = filepath.Join(rootdir, "/usr/share/bash-completion/completions/")
+	CompletersDir = filepath.Join(rootdir, snappyDir, "share/bash-completion/completions/")
```

Comment:
I wonder if `/var/lib/snapd/desktop/bas-completion/completions` isn't more appropriate here which has an upside of already being included in XDG_DATA_DIRS

Reply:
My problem with it is `desktop`. I do not really like to keep a name for historical reason. And I would prefer to migrate files from `/var/lib/snapd/desktop` to `/var/lib/snapd/share`.

Reply:
It now uses `/var/lib/snapd/desktop`.